spec/features/recipes_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
```

Comment:
- [ ] [OPTIONAL] Good job so far! Please fix linter errors without this kind of comment `# rubocop:disable ... ` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃

Reply:
Thanks for this feedback [Rizo](@mukhammadrizooff). You very observant 👍🏾